1.8/s2i/bin/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+exec /opt/app-root/gobinary $RUN_PARAMS
```

Comment:
Would be great to have described all variables that can be used in README.

Reply:
Added some docs for the IMPORT_URL, dropped the env var from run script as I'm not sure that it would be actually working/usable.